README.md

```diff
@@ -65,9 +66,9 @@ required packages:
 The following prerequisites are required when building the libraries for
 desktop platforms.
 
-- [OpenSSL](https://www.openssl.org/), needed for Realtime Database
+- [OpenSSL](https://www.openssl.org/), needed for Realtime Database and Cloud Firestore
 - [Protobuf](https://github.com/protocolbuffers/protobuf/blob/master/src/README.md),
-  needed for Remote Config
+  needed for Remote Config and Cloud Firestore
```

Comment:
Hmm, I'm not sure if we need `Protobuf`, can you clarify?

Reply:
I had to run `brew install protobuf` (and `sudo apt-get install protobuf-compiler` on Linux) when I first wanted to compile the firebase-cpp-sdk repo (although, now that I think about it, I built all the targets... perhaps if I had only built the `firebase_firestore` target alone it might have not been needed?)

Reply:
We do use `Protobuf` in the iOS repo (in integration tests), maybe that's the reason?

It would be best to temporarily uninstall `protobuf` (or rename the executable, etc.) and try again, can you please try that?

Reply:
I tried this, and the compilation was successful without it. So I'll update the readme